pepper-apis/studybuilder-cli/studies/lms/snippets/germline-consent-form-introduction.conf

```diff
@@ -0,0 +1,58 @@
+{
+    "titleTemplate": {
+     "TemplateType": "HTML",
+     "templateText": """
+                     $germline_intro
+                     """,
+      "variables": [
+        {
+          "name": "germline_intro",
+          "translations": [
+            {
+             "language": "en",
+             "text": "Introduction"
+            }
+          ]
+        }
+      ],
+    },
+   "bodyTemplate": {
+    "TemplateType": "HTML",
+    "templateText": """
+                    <p>$osteo_germline_consent_addendum_intro1</p>
```

Comment:
<p>$osteo_germline_consent_addendum_intro1</p> please change name osteo to lms or something else